src/noshow/dashboard/connection.py

```diff
@@ -66,6 +68,16 @@ def get_patient_list(_session: Session, date_input: date, top_n: int = 20) -> Li
         .group_by(ApiPrediction.patient_id)
         .order_by(func.max(ApiPrediction.prediction).desc())
         .limit(top_n)
+        .outerjoin(ApiPrediction.patient_relation)
+        # select rows where last_call_date is null, today or more than x months ago
+        .where(
+            (ApiPatient.last_call_date == None)  # noqa: E711
+            | (
+                ApiPatient.last_call_date
+                <= date.today() - pd.DateOffset(months=MUTE_PERIOD)
```

Comment:
The resulting type of this statement is a pandas Timestamp, not a date.

Maybe cleaner to use the datetime package, since pandas is not needed here, like:
`ApiPatient.last_call_date <= date.today() - timedelta(days=round(MUTE_PERIOD * 30.5))`